apps/taskman/jira_serializer.py

```diff
@@ -11,6 +11,89 @@
 """
 from rest_framework import serializers
 
+from apps.osim.workflow import WorkflowModel
+
+from .models import WORKFLOW_TO_JIRA_STATUS, JiraResolution, JiraStatus, Task
+
+
+class TaskSerializer(serializers.ModelSerializer):
```

Comment:
I am thinking about the task management use-cases and that they are very limited. The task creation should probably not eve be allowed through the API. It should be an automatic side-effect of the flaw creation. The possible task updates are also very limited - it is just promotion to the next state or rejection. Therefore I am not sure whether the full API makes much sense. The user of the API should IMO not even be aware of that there is a task in background. (s)he just works with the flaw and its status:workflow by the simple handles like promote.